wagtail/admin/ui/tables/pages.py

```diff
@@ -21,3 +29,52 @@ def get_value(self, instance):
 
 class PageStatusColumn(BaseColumn):
     cell_template_name = "wagtailadmin/pages/listing/_page_status_cell.html"
+
+
+class BulkActionsColumn(BaseColumn):
```

Comment:
We have a `wagtail.admin.ui.tables.BulkActionsCheckboxColumn` as of 25e753e64da74cba960c9c90b9d71b852b45404b. Perhaps this can be incorporated into that, or made into a subclass?